pages/Teams/Player_Profiles/player-profile-2.cfm

```diff
@@ -224,25 +224,32 @@
 	          </tbody>
 	        </table>
 		
-            <table class="bolder">
+             <cfif getPlayerAwards.recordCount EQ 0>
```

Comment:
I like where your head is at! This logic definitely works and could be used in different parts of our codebase so keep it in the back of your pocket.